manager/integration/tests/common.py

```diff
@@ -1684,7 +1684,10 @@ def node_default_tags():
 
     tag_mappings = {}
     for tags, node in zip(DEFAULT_TAGS, nodes):
-        assert len(node.disks) == 1
+        if DATA_ENGINE == "v2":
+            assert len(node.disks) == 2
+        else:
+            assert len(node.disks) == 1
```

Comment:
Could you leave a comment explaining why the len(node.disks) is 2 while enabling v2 data engine?

Reply:
Updated, thank you.